TACC/tup-ui
# Add plugin for rapid generation of staff pages - more design tweaks

## Overview

Changes after Design Review № 2.

## Related

- [TUP-548](https://jira.tacc.utexas.edu/browse/TUP-548)
- builds atop #253

## Changes

1. do not require Phone
2. ~~missing space between elements~~
    <sup>[user error caused by developer mistake](https://github.com/TACC/tup-ui/pull/253#issuecomment-1684356914) (see [1])</sup>
3. ~~reduce margin under headings (except in content-block)~~
    <sup>fixed in #253 at 71b3319</sup>
4. add Job Title help text to clarify "Manager" text goes there
5. make UT seal image bigger

## Testing

1. Save plugin instance with no phone. Confirm no error.
2. —
3. —
4. Review help text under "Job Title".
5. Save plugin instance with no image. Confirm UT seal is as wide as container.

## UI

### Before

![before](https://github.com/TACC/tup-ui/assets/62723358/d1e4795a-1d7f-456a-aad4-004476f20c74)

https://github.com/TACC/tup-ui/assets/62723358/99f04368-bf73-4dfa-85ab-589ea59ddf0b

### After

![after](https://github.com/TACC/tup-ui/assets/62723358/53814685-200f-41b4-a3ce-23705e29ef74)

https://github.com/TACC/tup-ui/assets/62723358/f1d1aa49-e0a7-49c4-9c17-44c9525633ae
